src/tests/functional/shared_test_classes/src/base/utils/generate_inputs.cpp

```diff
@@ -1008,6 +1008,14 @@ ov::runtime::Tensor generate(const
     return tensor;
 }
 
+ov::runtime::Tensor generate(const
+                             std::shared_ptr<ov::op::v1::BinaryConvolution>& node,
+                             size_t port,
+                             const ov::element::Type& elemType,
+                             const ov::Shape& targetShape) {
+    return ov::test::utils::create_and_fill_tensor(elemType, targetShape, 1, 0, 1, 7235346);
```

Comment:
Please use `InGenerateData` instead of these values

Reply:
deleted input generation for this test